doc/source/how-to-run-simulations.rst

```diff
@@ -1,291 +1,338 @@
+core functionality of the ``Simulation Engine`` can be used from within a Google Colab
+or Jupyter environment by means of `run_simulation
+<ref-api-flwr.html#flwr.simulation.run_simulation>`_.
+
+.. code-block:: python
+
+    from flwr.simulation import run_simulation
+
+    # Construct the ClientApp passing the client generation function
+    client_app = ClientApp(client_fn=client_fn)
+
+    # Create your ServerApp passing the server generation function
+    server_app = ServerApp(server_fn=server_fn)
+
+    run_simulation(
+        server_app=server_app,
+        client_app=client_app,
+        num_supernodes=10,  # equivalent to setting `num-supernodes` in the pyproject.toml
+    )
+
+With ``run_simulation``, you can also control the amount of resources for your
+``ClientApp`` instances. Do so by setting ``backend_config``. If unset, the default
+resources are assigned (i.e., 2xCPUs per ``ClientApp`` and no GPU).
+
+.. code-block:: python
+
+    run_simulation(
+        # ...
+        backend_config={"client_resources": {"num_cpus": 2, "num_gpus": 0.25}}
+    )
+
+Refer to the `30 minutes Federated AI Tutorial
+<https://colab.research.google.com/github/adap/flower/blob/main/examples/flower-in-30-minutes/tutorial.ipynb>`_
+for a complete example on how to run Flower Simulations in Colab.
 
 Multi-node Flower simulations
 -----------------------------
 
-Flower's ``VirtualClientEngine`` allows you to run FL simulations across multiple
-compute nodes. Before starting your multi-node simulation ensure that you:
-
-1. Have the same Python environment in all nodes.
-2. Have a copy of your code (e.g. your entire repo) in all nodes.
-3. Have a copy of your dataset in all nodes (more about this in :ref:`simulation
-   considerations <considerations-for-simulations>`)
-4. Pass ``ray_init_args={"address"="auto"}`` to `start_simulation
-   <ref-api-flwr.html#flwr.simulation.start_simulation>`_ so the ``VirtualClientEngine``
-   attaches to a running Ray instance.
-5. Start Ray on you head node: on the terminal type ``ray start --head``. This command
+Flower's ``Simulation Engine`` allows you to run FL simulations across multiple compute
+nodes. Before starting your multi-node simulation, ensure that you:
+
+1. Have the same Python environment on all nodes.
+2. Have a copy of your code (e.g., your entire repo) on all nodes.
+3. Have a copy of your dataset on all nodes. If you are using partitions from `Flower
+   Datasets <https://flower.ai/docs/datasets>`_, ensure they are the same.
+4. Start Ray on your head node: on the terminal, type ``ray start --head``. This command
    will print a few lines, one of which indicates how to attach other nodes to the head
    node.
-6. Attach other nodes to the head node: copy the command shown after starting the head
-   and execute it on terminal of a new node: for example ``ray start
-   --address='192.168.1.132:6379'``
+5. Attach other nodes to the head node: copy the command shown after starting the head
```

Comment:
Should we mention here that all the nodes should be discoverable by the head node?